docs/exchanges.md

```diff
@@ -33,3 +33,25 @@ To download data for the Kraken exchange, using `--dl-trades` is mandatory, othe
 ```shell
 $ pip3 install web3
 ```
+
+## Bittrex
+
+### Restricted markets
+
+Bittrex split its exchange into US and International versions.
+The International version has more pairs available, however the API always returns all pairs, so there is currently no automated way to detect if you're affected by the restriction.
+
+If you have restricted pairs in your whitelist, you'll get a warning message in the log on FreqTrade startup for each restricted pair.
```

Comment:
"RESTRICTED_MARKET" should be explicitly mentioned here, to let the user a clue which particular message should she look for.